test/cypress/dashboards/integration/Components/HTML/HTMLComponent.cy.js

```diff
@@ -34,4 +34,27 @@ describe('Updating HTML component.', () => {
             );
         })
     });
+
+    it('Should edit the HTML component with sidebar.', () => {
+        // Arrange- open sidebar
+        cy.toggleEditMode();
+        cy.openCellEditSidebar('#dashboard-2');
+
+        // Act- edit the HTML component
+        cy.get('.highcharts-dashboards-edit-accordion-header-btn')
+            .contains('span', 'HTML')
```

Comment:
Is it not easier to find elements and click by index? 

Eg `cy.get('.highcharts-dashboards-edit-accordion-header-btn').eq(4)` where 4 is the index of element.

Reply:
It probably is now but if we add anything to the demo in the future we break this test. Let's leave this slightly longer but more robust version